test/v2/unit/trading.ts

```diff
@@ -0,0 +1,74 @@
+import { coroutine as co } from 'bluebird';
+import * as should from 'should';
+import * as nock from 'nock';
+
+const Wallet = require('../../../src/v2/wallet');
+const TestV2BitGo = require('../../lib/test_bitgo');
+const common = require('../../../src/common');
+
+describe('V2 Trading', function() {
+  const microservicesUri = 'https://bitgo-microservices-endpoint.com';
```

Comment:
Please use an invalid tld so we don't inadvertently make requests here. `bitgo-microservices.example` might be a good choice.